acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/slashing/SlashingAcceptanceTest.java

```diff
@@ -0,0 +1,84 @@
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.slashing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.teku.bls.impl.blst.BlstSecretKey;
+import tech.pegasys.web3signer.core.service.http.ArtifactType;
+import tech.pegasys.web3signer.core.service.http.Eth2SigningRequestBody;
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.signer.SignerConfigurationBuilder;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+import tech.pegasys.web3signer.tests.AcceptanceTestBase;
+
+import java.nio.file.Path;
+import java.util.Random;
+
+import io.restassured.response.Response;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt64;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class SlashingAcceptanceTest extends AcceptanceTestBase {
+
+  private static final String SLASHING_DB_URL = System.getenv("SLASHING_DB_URL");
+  private static final String SLASHING_DB_USERNAME = System.getenv("SLASHING_DB_USERNAME");
+  private static final String SLASHING_DB_PASSWORD = System.getenv("SLASHING_DB_PASSWORD");
+
+  private static final MetadataFileHelpers metadataFileHelpers = new MetadataFileHelpers();
+
+  @BeforeAll
+  static void setup() {
+    Assumptions.assumeTrue(SLASHING_DB_URL != null, "Set SLASHING_DB_URL environment variable");
+    Assumptions.assumeTrue(
+        SLASHING_DB_USERNAME != null, "Set SLASHING_DB_USERNAME environment variable");
+    Assumptions.assumeTrue(
+        SLASHING_DB_PASSWORD != null, "Set SLASHING_DB_PASSWORD environment variable");
+  }
+
+  @Test
+  void cannotSignSameAttestationTwiceWhenSlashingIsEnabled(@TempDir Path testDirectory) {
+    final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
+    builder.withMode("eth2");
+    builder.withSlashingProtectionDbUrl(SLASHING_DB_URL);
+    builder.withSlashingProtectionDbUsername(SLASHING_DB_USERNAME);
+    builder.withSlashingProtectionDbPassword(SLASHING_DB_PASSWORD);
+    builder.withKeyStoreDirectory(testDirectory);
+
+    final BlstSecretKey secretKey = BlstSecretKey.generateNew(new Random());
+    final Path keyConfigFile = testDirectory.resolve("keyfile.yaml");
+    metadataFileHelpers.createUnencryptedYamlFileAt(
+        keyConfigFile, secretKey.toBytes().toHexString(), KeyType.BLS);
+
+    startSigner(builder.build());
+
+    final Bytes signingRoot = Bytes.fromHexString("0x01");
+    final Eth2SigningRequestBody request =
+        new Eth2SigningRequestBody(
+            signingRoot,
+            ArtifactType.ATTESTATION,
```

Comment:
not sure why this test is working as the attestation slashing protection hasn't been merged

Reply:
This is based on master, so attestations should now work